weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/interface.ts

```diff
@@ -78,6 +78,16 @@ export const spanToCallSchema = (
   project: string,
   span: SpanWithFeedback
 ): CallSchema => {
+  // This rawSpan construction fixes issues with crashed runs using the
```

Comment:
more graceful handling of bug in graph client where we are crashing before failing the run